superset-frontend/src/components/DatabaseSelector/index.tsx

```diff
@@ -87,7 +87,7 @@ export type DatabaseObject = {
 type SchemaValue = { label: string; value: string };
 
 export interface DatabaseSelectorProps {
-  db?: DatabaseObject;
+  db?: DatabaseObject | null | undefined;
```

Comment:
I understand why it is null, but why does it also need to be undefined?

Reply:
I think i was trying to appease ts in on the props. But I removed and no issue came about